icepyx/core/spatial.py

```diff
@@ -698,10 +698,10 @@ def fmt_for_CMR(self) -> str:
 
         """
         # CMR keywords: ['bounding_box', 'polygon']
-        if self._ext_type == "bounding_box":
-            cmr_extent = ",".join(map(str, self._spatial_ext))
+        if self.extent_type == "bounding_box":
+            return ",".join(map(str, self._spatial_ext))
```

Comment:
When there's an assignment like this, that tells me that something may be done to this variable after the conditional ends. Instead, `return` tells the reader right away that nothing further will happen.